src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -24,6 +24,9 @@ public partial class DllNotFoundException : System.TypeLoadException
         public DllNotFoundException(string message) { }
         public DllNotFoundException(string message, System.Exception inner) { }
     }
+    public partial struct RuntimeArgumentHandle
```

Comment:
Removed https://github.com/dotnet/standard/commit/455687a1c8f86a54fbd7da4f056333290411021c so we should remove it from this PR as well.